components/cost-management/staging/policies/propogate_namespace_labels.yaml

```
@@ -0,0 +1,55 @@
+---
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: propagate-cost-management-labels
+  annotations:
+    policies.kyverno.io/title: Propagate Cost-Center from Namespace
+    policies.kyverno.io/category: Cost Management
+    policies.kyverno.io/severity: low
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/description: >-
+      This policy ensures that: 
+      (1) The `cost-center` label is propagated from non-managed tenant namespaces to pods.
+      (2) The `insights_cost_management_optimizations` label is set to `true` on Pods to which the `cost-center` label was propagated.
+spec:
+  validationFailureAction: Audit
```

this is not needed here as the policy only contains a `mutate` rule. we need to use `spec.webhookConfiguration.failurePolicy: Ignore` instead

```suggestion
  webhookConfiguration:
    failurePolicy: Ignore
```

---

Sure. Changed this